lib/_emerge/BinpkgPrefetcher.py

```diff
@@ -67,12 +71,33 @@ def _verifier_exit(self, verifier):
 
             output_value = out.getvalue()
             if output_value:
-                self.scheduler.output(
-                    output_value,
-                    log_path=self.scheduler.fetch.log_file,
-                    background=self.background,
-                )
+                if injected_pkg is None:
+                    msg = ["Binary package is not usable:"]
+                    msg.extend("\t" + line for line in output_value.splitlines())
+                    self._elog("eerror", msg)
```

Comment:
We should probably call elog_process after this. It's called already in the pkg_pretend context reported in [bug 927632 comment 2](https://bugs.gentoo.org/927632#c2), but otherwise it's not called until later when it's time to install the package.